src/shared/nt-features/CreateFeatureForm.react.js

```diff
@@ -65,19 +69,26 @@ class CreateFeatureForm extends React.Component {
   }
 
   handleSubmit = e => {
+    e.preventDefault()
```

Comment:
order is matter. if there is sync function before `e.preventDefault()`, events will run a code before canceling them.